src/api.rs

```diff
@@ -89,6 +93,10 @@ impl Default for EncoderConfig {
 }
 
 impl EncoderConfig {
+  /// This is the preset set with width and height to 640 and 480 respectively,
```

Comment:
Maybe omit the details that could and will change and just mention that provides a default setting according to a speed value in the range `0-10`.

Mention what happens when you pass a number higher than `10`